utils/checks.stanza

```diff
@@ -1764,20 +1762,14 @@ public defn check-landpattern (component:JITXObject|JITXDef) :
 ; Runs landpattern checks recursively on all the lands in a module
 public defn check-all-landpatterns (module:JITXObject) : 
   inside pcb-module :
-    val components = to-tuple(component-instances(module))
-    val modules    = filter({not contains?(components, _)}, instances(module))
-    for component in components do :
-      check-landpattern(component)
-    for module in modules do :
-      check-all-landpatterns(module)
+    do(check-landpattern, component-instances(module))
```

Comment:
Better but far from optimum. If there is 100x the same LED or resistor, we don't want 100 times the same check.  We should group them by component definition. @ariel-hi do you know the introspection method to do that?